src/wazuh_db/wdb.h

```diff
@@ -826,6 +827,15 @@ int wdb_exec_stmt_send(sqlite3_stmt* stmt, int peer);
  */
 cJSON * wdb_exec_stmt(sqlite3_stmt * stmt);
 
+/**
+ * @brief Function to execute a SQL statement and save the result in a single JSON array without column name like:
+ *        ["string1","string2", ...]. The query should return only one column in every step.
```

Comment:
We can improve the example like:
["column_value_1","column_value_2", ...]

Reply:
Done!